src/NServiceBus.Persistence.AzureStorage/Outbox/OutboxRecord.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace NServiceBus.Persistence.AzureStorage
+{
+    using System;
+    using System.Collections.Generic;
+    using Outbox;
+    using Newtonsoft.Json;
+    using Microsoft.Azure.Cosmos.Table;
+
+    class OutboxRecord : TableEntity
+    {
+        [IgnoreProperty]
```

Comment:
A comment on why we ignore the Id property (name colliding with Cosmos DB built-in property) would be good to have.